src/log.rs

```diff
@@ -28,15 +38,29 @@ pub fn show(mut connection: Connection, verbose: bool) {
                 } else {
                     Time::Local(Local::now().naive_local())
                 };
-                let qos = format::qos(publish.qos);
                 let topic = publish.topic;
                 let size = publish.payload.len();
                 let payload = Payload::unlimited(publish.payload.into());
-                println!("{time:12} QoS:{qos:11} {topic:50} Payload({size:>3}): {payload}");
+
```

Comment:
Really minor nitpick: As soon as I need empty lines in my code it suggests the code is getting more complex and might need to be moved out into a function. This isn't as complex so no real need to split it into groups separated by empty lines :innocent: 
```suggestion
```